database/migrations/default/1740674457123_feb-2025-lookup-table-updates/up.sql

```diff
@@ -0,0 +1,11 @@
+delete from lookups.agency where id = 10101;
+insert into lookups.agency (id, label, source) values (10233, 'CITY OF DUNCANVILLE', 'cris');
+insert into lookups.agency (id, label, source) values (10234, 'CITY OF FORNEY', 'cris');
+insert into lookups.agency (id, label, source) values (10235, 'ROBINSON VOLUNTEER FIRE DEPARTMENT', 'cris');
+insert into lookups.agency (id, label, source) values (3365, 'BONHAM ISD POLICE DEPARTMENT', 'cris');
+insert into lookups.agency (id, label, source) values (3366, 'ROUND ROCK ISD POLICE DEPARTMENT', 'cris');
```

Comment:
it looks like this was the culprit, based on my attempt to run the `down` migration after importing the latest data.